### PR TITLE
Agreements

### PR DESCRIPTION
**Pull request by shereefb**
Updates guide with agreements from the retreat. 

@tannerwelsh @carlabagdonas would love your eyes on this before I merge.

### CODE REVIEW

File: Guiding-Principles/agreements.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
+# Tribe Agreements
+
+## Org Space & Tribe Space
+
+There are two distinct and complementary 'modes' or 'spaces' that we operate in as partners. **Org Space** concerns itself with getting the work done. Moving the company towards its purpose tactically and strategically. Our [Operating Model](Operating-Model) spells out the details of our **Org Space** agreements. When operating in **Org Space** we intentionally put our personal tensions aside and step into our roles and function from there.
+
+We are, however, much more than our roles. In our work together we are affected as people, not just as roles. In the course of working with each other, we impact each other. Toes get stepped on, feelings get hurt, friendships start and end. Misunderstandings. Attraction. Power struggles. You name it.
+
+Our Tribe agreements are our attempt to deliberately address our interpersonal relationships, and care for them. In doing so, we are more able to fulfill our organizations' purpose.
```
Comment (tannerwelsh) [inline Guiding-Principles/agreements.md]: 👍 love it. Minor typo: `organization's` not `organizations'`

File: Guiding-Principles/agreements.md [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+
+Another good use of sharing a reflection, is with regards to the Five Personal Practices. Because these practices are interpersonal, giving and receiving reflections from each other on our progress and the areas where we need to practice more supports them in staying alive.
+
+If you tried to clear a withhold with someone and failed, use our [Conflict Resloution Process][conflict-process] to resolve it.
+
+### Hold Others Accountable
+
+Our agreements are meaningless if we don't hold ourselves and each other accountable to them. It is crucial that we step up to violated promises, broken commitments and bad behavior. In doing so, we maintain our integrity, and support each other lovingly and firmly in doing the same.
+
+## Five Interpersonal Practices
```
Comment (tannerwelsh) [inline Guiding-Principles/agreements.md]: Consider changing the name to just "Five Practices", and a description of:

```
We agree to engage in the following aspirational practices. By definition...
```

The "interpersonal" part seems redundant, and not super applicable to "Whole Heartedness" or "Radical Responsibility"

**Comment by tannerwelsh** [comment]
Looks good, thanks @shereefb! I have plenty of nit-picky edits I could propose, but for now I'm fine to merge as is.

**Comment by carlabagdonas** [comment]
Alright, it's still difficult for me to figure out how to comment in-line, so I'll leave my thoughts here:

**Content:**

I have some reservations about these paragraphs:

If you have feedback for a partner, the best and only place for this feedback is to give it to them directly. **If that's not working, get others to join the conversation.**
--> it's not clear to me what this means. do i get a clearing coach? get another partner to join me in talking to that person? talk to another partner about the trouble i'm having? it's not clear that you're not suggesting triangulation as a solution if non-triangulation doesn't work.

If you have feedback for supporting staff, or other stakeholders, do your best to give it directly to them. **If that's not possible/efficient/effective it's OK to talk about them with another partner.**
--> I disagree strongly with this. Support staff who are working with us on a consistent basis (e.g. justin, lavinia, sharyn) should be on board with the same agreements and practices that partners are, and we should not be giving ourselves license to triangulate about support staff differently than we do about partners. I am very wary of creating "class divisions" within our company and worry we've already started down this path.

Suggest re-naming "Five Interpersonal Practices" to "Five Practices" and re-naming the practices themselves as follows:
- Be kind
- Be wholehearted
- Take radical responsibility
- Be real and vulnerable
- Practice resisting and overcoming oppression

The "Practice [noun]" version feels watered down and linguistically complicated.


**Grammatical**:

**missing a period** For this reason, every partner takes on a secondary purpose: to co-create a thriving culture for our team that brings out the best in us

**consistent capitals**: Act in Integrity, Don't Triangulate, Clear Personal Tensions [suggest removing "swiftly and directly"], Hold Others Accountable

**consistent naming of "practices"**: Another good use of sharing a reflection, is with regards to the **Five Personal Practices**. Because these practices are interpersonal, giving and receiving reflections from each other on our progress and the areas where we need to practice more supports them in staying alive. / later referred to as **Interpersonal Practices**
--> I like Tanner's suggestion of just calling them "practices"

**Pull request merged**